src/languages/en-US.ts

```diff
@@ -602,6 +602,16 @@ export default class extends Language {
 				their marks in a horizontal, vertical, or diagonal row wins the game.`
 		}),
 
+		/**
+		 * ###################
```

Comment:
Always aligned to the title's length:
```suggestion
		 * ################
```